test/check-realms

```diff
@@ -112,6 +112,10 @@ class TestRealms(MachineCase):
         b.click(".realms-op-cancel")
         b.wait_popdown("realms-op")
 
+        # free the ipa domain
```

Comment:
The term "domain" is confusing here, since it might also refer to the FreeIPA concept.